config/cluster-baremetal-operator-hostedcluster/kustomization.yaml

```diff
@@ -1,7 +1,6 @@
 commonAnnotations:
   include.release.openshift.io/ibm-cloud-managed: "true"
   include.release.openshift.io/hypershift: "true"
-  include.release.openshift.io/single-node-developer: "true"
```

Comment:
Does this mean the delete manifest should also have SNO inclusion?

As in, previous SNO clusters will have this Deployment, post-change they won't and we may run into the same problem on SNO clusters?

Reply:
No, not by this PR. But yeah, you are right. I am confused _why_ we see the SNO updates stuck though. I'd say that SNO must have had deployed the `manifests/0000_31_cluster-baremetal-operator_06_deployment-hostedcluster.yaml` until now because the SNO annotation was not removed in the earlier PR. But then nothing should change for SNO clusters and still we are seeing them stuck in update. That's a bit surprising to me.

Let's see what the SNO payload jobs say

Reply:
SNO payload jobs look good